src/mame/layout/smiconk.lay

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="overlay">
+		<rect>
+			<bounds x="0" y="0" width="1" height="240" />
+			<color red="1" green="1" blue="0.2" />
```

Comment:
If you’re creating a flat quad, you don’t need to use bounds – taking the default unit square keeps things simpler.

Reply:
Right, done